packages/shorebird_cli/shorebird.yaml

```diff
@@ -0,0 +1,3 @@
+# This file is used to configure the Shorebird CLI.
+# Learn more at https://shorebird.dev
+product_id: b1c05854-4997-42ed-a2a5-315ecc35f093
```

Comment:
Did you mean to check this in?

Reply:
nope will remove thanks for catching that!